package.json

```diff
@@ -76,15 +164,17 @@
     "time-grunt": "~1.2.2"
   },
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=0.10.0",
+    "yarn": ">= 1.0.0"
   },
   "scripts": {
     "update-webdriver": "node node_modules/.bin/webdriver-manager update",
     "commit": "npm-scripts-config commit",
     "commitmsg": "npm-scripts-config commitmsg",
     "version": "npm-scripts-config version && npm-bower-sync-ver && git add bower.json",
     "postversion": "git push && git push --tags",
-    "preview-changelog": "npm-scripts-config preview-changelog"
+    "preview-changelog": "npm-scripts-config preview-changelog",
+    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'client/bower_components', 'junction') } catch (e) { }\""
```

Comment:
Is there any reason of doing that kind of thing?

Reply:
This line was added by bower-away and I kept it because I found it useful:
It creates a link from `node_modules/@bower_components` to `client/bower_components`, preventing us from having to change all references to bower_components in the code.
That's why I also added the symbolic link to the .gitignore, so that git does not try to push the link file (it's a file that links to a folder, but still a file to git).